<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="20" passed="0" failed="3" skipped="17">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-05-11T15:27:11 GMT" name="book-flight-module" finished-at="2023-05-11T15:30:27 GMT" duration-ms="195456">
    <groups>
    </groups>
    <test started-at="2023-05-11T15:27:11 GMT" name="book-flight-1" finished-at="2023-05-11T15:30:13 GMT" duration-ms="182372">
      <class name="com.newtours.tests.BookFlightTest">
        <test-method is-config="true" signature="setupDriver()[pri:0, instance:com.newtours.tests.BookFlightTest@5c90e579]" started-at="2023-05-11T15:27:11 GMT" name="setupDriver" finished-at="2023-05-11T15:30:13 GMT" duration-ms="182299" status="FAIL">
          <exception class="org.openqa.selenium.SessionNotCreatedException">
            <message>
              <![CDATA[Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: '157ba0768012', ip: '172.19.0.10'
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '5.15.49-linuxkit', java.version: '17.0.6'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Command: [null, newSession {capabilities=[Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}], desiredCapabilities=Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}}]
Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: '157ba0768012', ip: '172.19.0.10'
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '5.15.49-linuxkit', java.version: '17.0.6'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Command: [null, newSession {capabilities=[Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}], desiredCapabilities=Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}}]
Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:561)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:229)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:157)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:139)
at com.tests.BaseTest.setupDriver(BaseTest.java:46)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:644)
at org.testng.TestRunner.beforeRun(TestRunner.java:633)
at org.testng.TestRunner.run(TestRunner.java:595)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.RuntimeException: NettyHttpHandler request execution error
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.makeCall(NettyHttpHandler.java:76)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:56)
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.execute(NettyHttpHandler.java:49)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:56)
at org.openqa.selenium.remote.http.netty.NettyClient.execute(NettyClient.java:99)
at org.openqa.selenium.remote.tracing.TracedHttpClient.execute(TracedHttpClient.java:55)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:124)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:106)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:67)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:165)
at org.openqa.selenium.remote.TracedCommandExecutor.execute(TracedCommandExecutor.java:51)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
... 22 more
Caused by: java.util.concurrent.ExecutionException: java.util.concurrent.TimeoutException: Request timeout to hub/172.19.0.2:4444 after 180000 ms
at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396)
at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2096)
at org.asynchttpclient.netty.NettyResponseFuture.get(NettyResponseFuture.java:206)
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.makeCall(NettyHttpHandler.java:59)
... 35 more
Caused by: java.util.concurrent.TimeoutException: Request timeout to hub/172.19.0.2:4444 after 180000 ms
at org.asynchttpclient.netty.timeout.TimeoutTimerTask.expire(TimeoutTimerTask.java:43)
at org.asynchttpclient.netty.timeout.RequestTimeoutTimerTask.run(RequestTimeoutTimerTask.java:50)
at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
... 1 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.SessionNotCreatedException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupDriver -->
        <test-method is-config="true" signature="setupParameters(java.lang.String,java.lang.String)[pri:0, instance:com.newtours.tests.BookFlightTest@5c90e579]" started-at="2023-05-11T15:30:13 GMT" name="setupParameters" finished-at="2023-05-11T15:30:13 GMT" depends-on-methods="com.tests.BaseTest.setupDriver" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupParameters -->
        <test-method signature="registrationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@5c90e579]" started-at="2023-05-11T15:30:13 GMT" name="registrationPage" finished-at="2023-05-11T15:30:13 GMT" duration-ms="0" status="SKIP">
          <exception class="org.openqa.selenium.SessionNotCreatedException">
            <message>
              <![CDATA[Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: '157ba0768012', ip: '172.19.0.10'
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '5.15.49-linuxkit', java.version: '17.0.6'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Command: [null, newSession {capabilities=[Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}], desiredCapabilities=Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}}]
Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: '157ba0768012', ip: '172.19.0.10'
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '5.15.49-linuxkit', java.version: '17.0.6'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Command: [null, newSession {capabilities=[Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}], desiredCapabilities=Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}}]
Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:561)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:229)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:157)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:139)
at com.tests.BaseTest.setupDriver(BaseTest.java:46)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:644)
at org.testng.TestRunner.beforeRun(TestRunner.java:633)
at org.testng.TestRunner.run(TestRunner.java:595)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.RuntimeException: NettyHttpHandler request execution error
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.makeCall(NettyHttpHandler.java:76)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:56)
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.execute(NettyHttpHandler.java:49)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:56)
at org.openqa.selenium.remote.http.netty.NettyClient.execute(NettyClient.java:99)
at org.openqa.selenium.remote.tracing.TracedHttpClient.execute(TracedHttpClient.java:55)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:124)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:106)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:67)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:165)
at org.openqa.selenium.remote.TracedCommandExecutor.execute(TracedCommandExecutor.java:51)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
... 22 more
Caused by: java.util.concurrent.ExecutionException: java.util.concurrent.TimeoutException: Request timeout to hub/172.19.0.2:4444 after 180000 ms
at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396)
at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2096)
at org.asynchttpclient.netty.NettyResponseFuture.get(NettyResponseFuture.java:206)
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.makeCall(NettyHttpHandler.java:59)
... 35 more
Caused by: java.util.concurrent.TimeoutException: Request timeout to hub/172.19.0.2:4444 after 180000 ms
at org.asynchttpclient.netty.timeout.TimeoutTimerTask.expire(TimeoutTimerTask.java:43)
at org.asynchttpclient.netty.timeout.RequestTimeoutTimerTask.run(RequestTimeoutTimerTask.java:50)
at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
... 1 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.SessionNotCreatedException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registrationPage -->
        <test-method signature="registrationConfirmationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@5c90e579]" started-at="2023-05-11T15:30:13 GMT" name="registrationConfirmationPage" finished-at="2023-05-11T15:30:13 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.registrationPage" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.registrationConfirmationPage() on instance com.newtours.tests.BookFlightTest@5c90e579 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.registrationPage() on instance com.newtours.tests.BookFlightTest@5c90e579]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.registrationConfirmationPage() on instance com.newtours.tests.BookFlightTest@5c90e579 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.registrationPage() on instance com.newtours.tests.BookFlightTest@5c90e579]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registrationConfirmationPage -->
        <test-method signature="flightConfirmationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@5c90e579]" started-at="2023-05-11T15:30:13 GMT" name="flightConfirmationPage" finished-at="2023-05-11T15:30:13 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.findFlightPage" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.flightConfirmationPage() on instance com.newtours.tests.BookFlightTest@5c90e579 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@5c90e579]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.flightConfirmationPage() on instance com.newtours.tests.BookFlightTest@5c90e579 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@5c90e579]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flightConfirmationPage -->
        <test-method signature="findFlightPage()[pri:0, instance:com.newtours.tests.BookFlightTest@5c90e579]" started-at="2023-05-11T15:30:13 GMT" name="findFlightPage" finished-at="2023-05-11T15:30:13 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.flightDetailsPage" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@5c90e579 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@5c90e579]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@5c90e579 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@5c90e579]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- findFlightPage -->
        <test-method signature="flightDetailsPage()[pri:0, instance:com.newtours.tests.BookFlightTest@5c90e579]" started-at="2023-05-11T15:30:13 GMT" name="flightDetailsPage" finished-at="2023-05-11T15:30:13 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.registrationConfirmationPage" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@5c90e579 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.registrationConfirmationPage() on instance com.newtours.tests.BookFlightTest@5c90e579]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@5c90e579 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.registrationConfirmationPage() on instance com.newtours.tests.BookFlightTest@5c90e579]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flightDetailsPage -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.newtours.tests.BookFlightTest@5c90e579]" started-at="2023-05-11T15:30:13 GMT" name="quitDriver" finished-at="2023-05-11T15:30:13 GMT" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
      </class> <!-- com.newtours.tests.BookFlightTest -->
    </test> <!-- book-flight-1 -->
    <test started-at="2023-05-11T15:27:11 GMT" name="book-flight-3" finished-at="2023-05-11T15:30:26 GMT" duration-ms="195392">
      <class name="com.newtours.tests.BookFlightTest">
        <test-method is-config="true" signature="setupDriver()[pri:0, instance:com.newtours.tests.BookFlightTest@1d8bd0de]" started-at="2023-05-11T15:27:11 GMT" name="setupDriver" finished-at="2023-05-11T15:27:26 GMT" duration-ms="14756" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupDriver -->
        <test-method is-config="true" signature="setupParameters(java.lang.String,java.lang.String)[pri:0, instance:com.newtours.tests.BookFlightTest@1d8bd0de]" started-at="2023-05-11T15:27:26 GMT" name="setupParameters" finished-at="2023-05-11T15:27:26 GMT" depends-on-methods="com.tests.BaseTest.setupDriver" duration-ms="83" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[$1753 USD]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupParameters -->
        <test-method signature="registrationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@1d8bd0de]" started-at="2023-05-11T15:27:26 GMT" name="registrationPage" finished-at="2023-05-11T15:30:26 GMT" duration-ms="180158" status="FAIL">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[java.util.concurrent.TimeoutException
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '5.15.49-linuxkit', java.version: '17.0.6'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Command: [a54ac6f84b9cf86153deb178470f19c2, get {url=https://vins-udemy.s3.amazonaws.com/docker/docker-book-flight.html}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 112.0.5615.165, chrome: {chromedriverVersion: 112.0.5615.49 (bd2a7bcb881c..., userDataDir: /tmp/.com.google.Chrome.IuGMoU}, goog:chromeOptions: {debuggerAddress: localhost:33697}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: LINUX, proxy: Proxy(), se:bidiEnabled: false, se:cdp: ws://172.19.0.5:4444/sessio..., se:cdpVersion: 112.0.5615.165, se:vnc: ws://172.19.0.5:4444/sessio..., se:vncEnabled: true, se:vncLocalAddress: ws://172.19.0.5:7900, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: a54ac6f84b9cf86153deb178470f19c2]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: java.util.concurrent.TimeoutException
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '5.15.49-linuxkit', java.version: '17.0.6'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Command: [a54ac6f84b9cf86153deb178470f19c2, get {url=https://vins-udemy.s3.amazonaws.com/docker/docker-book-flight.html}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 112.0.5615.165, chrome: {chromedriverVersion: 112.0.5615.49 (bd2a7bcb881c..., userDataDir: /tmp/.com.google.Chrome.IuGMoU}, goog:chromeOptions: {debuggerAddress: localhost:33697}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: LINUX, proxy: Proxy(), se:bidiEnabled: false, se:cdp: ws://172.19.0.5:4444/sessio..., se:cdpVersion: 112.0.5615.165, se:vnc: ws://172.19.0.5:4444/sessio..., se:vncEnabled: true, se:vncLocalAddress: ws://172.19.0.5:7900, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: a54ac6f84b9cf86153deb178470f19c2
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.makeCall(NettyHttpHandler.java:65)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:56)
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.execute(NettyHttpHandler.java:49)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:56)
at org.openqa.selenium.remote.http.netty.NettyClient.execute(NettyClient.java:99)
at org.openqa.selenium.remote.tracing.TracedHttpClient.execute(TracedHttpClient.java:55)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:190)
at org.openqa.selenium.remote.TracedCommandExecutor.execute(TracedCommandExecutor.java:51)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:297)
at com.newtours.pages.RegistrationPage.goTo(RegistrationPage.java:42)
at com.newtours.tests.BookFlightTest.registrationPage(BookFlightTest.java:25)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.util.concurrent.TimeoutException
at java.base/java.util.concurrent.CompletableFuture.timedGet(CompletableFuture.java:1960)
at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2095)
at org.asynchttpclient.netty.NettyResponseFuture.get(NettyResponseFuture.java:206)
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.makeCall(NettyHttpHandler.java:59)
... 35 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registrationPage -->
        <test-method signature="registrationConfirmationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@1d8bd0de]" started-at="2023-05-11T15:30:26 GMT" name="registrationConfirmationPage" finished-at="2023-05-11T15:30:26 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.registrationPage" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.registrationConfirmationPage() on instance com.newtours.tests.BookFlightTest@1d8bd0de depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.registrationPage() on instance com.newtours.tests.BookFlightTest@1d8bd0de]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.registrationConfirmationPage() on instance com.newtours.tests.BookFlightTest@1d8bd0de depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.registrationPage() on instance com.newtours.tests.BookFlightTest@1d8bd0de]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registrationConfirmationPage -->
        <test-method signature="flightDetailsPage()[pri:0, instance:com.newtours.tests.BookFlightTest@1d8bd0de]" started-at="2023-05-11T15:30:26 GMT" name="flightDetailsPage" finished-at="2023-05-11T15:30:26 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.registrationConfirmationPage" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@1d8bd0de depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.registrationConfirmationPage() on instance com.newtours.tests.BookFlightTest@1d8bd0de]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@1d8bd0de depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.registrationConfirmationPage() on instance com.newtours.tests.BookFlightTest@1d8bd0de]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flightDetailsPage -->
        <test-method signature="flightConfirmationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@1d8bd0de]" started-at="2023-05-11T15:30:26 GMT" name="flightConfirmationPage" finished-at="2023-05-11T15:30:26 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.findFlightPage" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.flightConfirmationPage() on instance com.newtours.tests.BookFlightTest@1d8bd0de depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@1d8bd0de]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.flightConfirmationPage() on instance com.newtours.tests.BookFlightTest@1d8bd0de depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@1d8bd0de]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flightConfirmationPage -->
        <test-method signature="findFlightPage()[pri:0, instance:com.newtours.tests.BookFlightTest@1d8bd0de]" started-at="2023-05-11T15:30:26 GMT" name="findFlightPage" finished-at="2023-05-11T15:30:26 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.flightDetailsPage" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@1d8bd0de depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@1d8bd0de]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@1d8bd0de depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@1d8bd0de]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- findFlightPage -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.newtours.tests.BookFlightTest@1d8bd0de]" started-at="2023-05-11T15:30:26 GMT" name="quitDriver" finished-at="2023-05-11T15:30:26 GMT" duration-ms="319" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
      </class> <!-- com.newtours.tests.BookFlightTest -->
    </test> <!-- book-flight-3 -->
    <test started-at="2023-05-11T15:27:11 GMT" name="book-flight-2" finished-at="2023-05-11T15:30:26 GMT" duration-ms="195423">
      <class name="com.newtours.tests.BookFlightTest">
        <test-method is-config="true" signature="setupDriver()[pri:0, instance:com.newtours.tests.BookFlightTest@2a266d09]" started-at="2023-05-11T15:27:11 GMT" name="setupDriver" finished-at="2023-05-11T15:27:26 GMT" duration-ms="14759" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupDriver -->
        <test-method is-config="true" signature="setupParameters(java.lang.String,java.lang.String)[pri:0, instance:com.newtours.tests.BookFlightTest@2a266d09]" started-at="2023-05-11T15:27:26 GMT" name="setupParameters" finished-at="2023-05-11T15:27:26 GMT" depends-on-methods="com.tests.BaseTest.setupDriver" duration-ms="83" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[$1169 USD]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupParameters -->
        <test-method signature="registrationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@2a266d09]" started-at="2023-05-11T15:27:26 GMT" name="registrationPage" finished-at="2023-05-11T15:30:26 GMT" duration-ms="180158" status="FAIL">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[java.util.concurrent.TimeoutException
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '5.15.49-linuxkit', java.version: '17.0.6'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Command: [2be6b81c220bb32b28bfdda3aaa7fa76, get {url=https://vins-udemy.s3.amazonaws.com/docker/docker-book-flight.html}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 112.0.5615.165, chrome: {chromedriverVersion: 112.0.5615.49 (bd2a7bcb881c..., userDataDir: /tmp/.com.google.Chrome.auJ1vU}, goog:chromeOptions: {debuggerAddress: localhost:36009}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: LINUX, proxy: Proxy(), se:bidiEnabled: false, se:cdp: ws://172.19.0.7:4444/sessio..., se:cdpVersion: 112.0.5615.165, se:vnc: ws://172.19.0.7:4444/sessio..., se:vncEnabled: true, se:vncLocalAddress: ws://172.19.0.7:7900, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 2be6b81c220bb32b28bfdda3aaa7fa76]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: java.util.concurrent.TimeoutException
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '5.15.49-linuxkit', java.version: '17.0.6'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Command: [2be6b81c220bb32b28bfdda3aaa7fa76, get {url=https://vins-udemy.s3.amazonaws.com/docker/docker-book-flight.html}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 112.0.5615.165, chrome: {chromedriverVersion: 112.0.5615.49 (bd2a7bcb881c..., userDataDir: /tmp/.com.google.Chrome.auJ1vU}, goog:chromeOptions: {debuggerAddress: localhost:36009}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: LINUX, proxy: Proxy(), se:bidiEnabled: false, se:cdp: ws://172.19.0.7:4444/sessio..., se:cdpVersion: 112.0.5615.165, se:vnc: ws://172.19.0.7:4444/sessio..., se:vncEnabled: true, se:vncLocalAddress: ws://172.19.0.7:7900, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 2be6b81c220bb32b28bfdda3aaa7fa76
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.makeCall(NettyHttpHandler.java:65)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:56)
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.execute(NettyHttpHandler.java:49)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:56)
at org.openqa.selenium.remote.http.netty.NettyClient.execute(NettyClient.java:99)
at org.openqa.selenium.remote.tracing.TracedHttpClient.execute(TracedHttpClient.java:55)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:190)
at org.openqa.selenium.remote.TracedCommandExecutor.execute(TracedCommandExecutor.java:51)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:297)
at com.newtours.pages.RegistrationPage.goTo(RegistrationPage.java:42)
at com.newtours.tests.BookFlightTest.registrationPage(BookFlightTest.java:25)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.util.concurrent.TimeoutException
at java.base/java.util.concurrent.CompletableFuture.timedGet(CompletableFuture.java:1960)
at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2095)
at org.asynchttpclient.netty.NettyResponseFuture.get(NettyResponseFuture.java:206)
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.makeCall(NettyHttpHandler.java:59)
... 35 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registrationPage -->
        <test-method signature="registrationConfirmationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@2a266d09]" started-at="2023-05-11T15:30:26 GMT" name="registrationConfirmationPage" finished-at="2023-05-11T15:30:26 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.registrationPage" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.registrationConfirmationPage() on instance com.newtours.tests.BookFlightTest@2a266d09 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.registrationPage() on instance com.newtours.tests.BookFlightTest@2a266d09]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.registrationConfirmationPage() on instance com.newtours.tests.BookFlightTest@2a266d09 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.registrationPage() on instance com.newtours.tests.BookFlightTest@2a266d09]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registrationConfirmationPage -->
        <test-method signature="flightDetailsPage()[pri:0, instance:com.newtours.tests.BookFlightTest@2a266d09]" started-at="2023-05-11T15:30:26 GMT" name="flightDetailsPage" finished-at="2023-05-11T15:30:26 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.registrationConfirmationPage" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@2a266d09 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.registrationConfirmationPage() on instance com.newtours.tests.BookFlightTest@2a266d09]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@2a266d09 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.registrationConfirmationPage() on instance com.newtours.tests.BookFlightTest@2a266d09]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flightDetailsPage -->
        <test-method signature="flightConfirmationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@2a266d09]" started-at="2023-05-11T15:30:26 GMT" name="flightConfirmationPage" finished-at="2023-05-11T15:30:26 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.findFlightPage" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.flightConfirmationPage() on instance com.newtours.tests.BookFlightTest@2a266d09 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@2a266d09]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.flightConfirmationPage() on instance com.newtours.tests.BookFlightTest@2a266d09 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@2a266d09]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flightConfirmationPage -->
        <test-method signature="findFlightPage()[pri:0, instance:com.newtours.tests.BookFlightTest@2a266d09]" started-at="2023-05-11T15:30:26 GMT" name="findFlightPage" finished-at="2023-05-11T15:30:26 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.flightDetailsPage" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@2a266d09 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@2a266d09]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@2a266d09 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@2a266d09]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- findFlightPage -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.newtours.tests.BookFlightTest@2a266d09]" started-at="2023-05-11T15:30:26 GMT" name="quitDriver" finished-at="2023-05-11T15:30:26 GMT" duration-ms="349" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
      </class> <!-- com.newtours.tests.BookFlightTest -->
    </test> <!-- book-flight-2 -->
    <test started-at="2023-05-11T15:27:11 GMT" name="book-flight-4" finished-at="2023-05-11T15:30:27 GMT" duration-ms="195451">
      <class name="com.newtours.tests.BookFlightTest">
        <test-method is-config="true" signature="setupDriver()[pri:0, instance:com.newtours.tests.BookFlightTest@36b4fe2a]" started-at="2023-05-11T15:27:11 GMT" name="setupDriver" finished-at="2023-05-11T15:27:26 GMT" duration-ms="14756" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupDriver -->
        <test-method is-config="true" signature="setupParameters(java.lang.String,java.lang.String)[pri:0, instance:com.newtours.tests.BookFlightTest@36b4fe2a]" started-at="2023-05-11T15:27:26 GMT" name="setupParameters" finished-at="2023-05-11T15:27:26 GMT" depends-on-methods="com.tests.BaseTest.setupDriver" duration-ms="83" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[4]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[$2338 USD]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupParameters -->
        <test-method signature="registrationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@36b4fe2a]" started-at="2023-05-11T15:27:26 GMT" name="registrationPage" finished-at="2023-05-11T15:30:26 GMT" duration-ms="180158" status="FAIL">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[java.util.concurrent.TimeoutException
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '5.15.49-linuxkit', java.version: '17.0.6'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Command: [156f1ae6b876c3da6a919c7f7696d7ee, get {url=https://vins-udemy.s3.amazonaws.com/docker/docker-book-flight.html}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 112.0.5615.165, chrome: {chromedriverVersion: 112.0.5615.49 (bd2a7bcb881c..., userDataDir: /tmp/.com.google.Chrome.3Yy2lV}, goog:chromeOptions: {debuggerAddress: localhost:43253}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: LINUX, proxy: Proxy(), se:bidiEnabled: false, se:cdp: ws://172.19.0.4:4444/sessio..., se:cdpVersion: 112.0.5615.165, se:vnc: ws://172.19.0.4:4444/sessio..., se:vncEnabled: true, se:vncLocalAddress: ws://172.19.0.4:7900, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 156f1ae6b876c3da6a919c7f7696d7ee]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: java.util.concurrent.TimeoutException
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '5.15.49-linuxkit', java.version: '17.0.6'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Command: [156f1ae6b876c3da6a919c7f7696d7ee, get {url=https://vins-udemy.s3.amazonaws.com/docker/docker-book-flight.html}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 112.0.5615.165, chrome: {chromedriverVersion: 112.0.5615.49 (bd2a7bcb881c..., userDataDir: /tmp/.com.google.Chrome.3Yy2lV}, goog:chromeOptions: {debuggerAddress: localhost:43253}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: LINUX, proxy: Proxy(), se:bidiEnabled: false, se:cdp: ws://172.19.0.4:4444/sessio..., se:cdpVersion: 112.0.5615.165, se:vnc: ws://172.19.0.4:4444/sessio..., se:vncEnabled: true, se:vncLocalAddress: ws://172.19.0.4:7900, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 156f1ae6b876c3da6a919c7f7696d7ee
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.makeCall(NettyHttpHandler.java:65)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:56)
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.execute(NettyHttpHandler.java:49)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:56)
at org.openqa.selenium.remote.http.netty.NettyClient.execute(NettyClient.java:99)
at org.openqa.selenium.remote.tracing.TracedHttpClient.execute(TracedHttpClient.java:55)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:190)
at org.openqa.selenium.remote.TracedCommandExecutor.execute(TracedCommandExecutor.java:51)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:297)
at com.newtours.pages.RegistrationPage.goTo(RegistrationPage.java:42)
at com.newtours.tests.BookFlightTest.registrationPage(BookFlightTest.java:25)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.util.concurrent.TimeoutException
at java.base/java.util.concurrent.CompletableFuture.timedGet(CompletableFuture.java:1960)
at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2095)
at org.asynchttpclient.netty.NettyResponseFuture.get(NettyResponseFuture.java:206)
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.makeCall(NettyHttpHandler.java:59)
... 35 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registrationPage -->
        <test-method signature="registrationConfirmationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@36b4fe2a]" started-at="2023-05-11T15:30:26 GMT" name="registrationConfirmationPage" finished-at="2023-05-11T15:30:26 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.registrationPage" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.registrationConfirmationPage() on instance com.newtours.tests.BookFlightTest@36b4fe2a depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.registrationPage() on instance com.newtours.tests.BookFlightTest@36b4fe2a]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.registrationConfirmationPage() on instance com.newtours.tests.BookFlightTest@36b4fe2a depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.registrationPage() on instance com.newtours.tests.BookFlightTest@36b4fe2a]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registrationConfirmationPage -->
        <test-method signature="flightDetailsPage()[pri:0, instance:com.newtours.tests.BookFlightTest@36b4fe2a]" started-at="2023-05-11T15:30:26 GMT" name="flightDetailsPage" finished-at="2023-05-11T15:30:26 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.registrationConfirmationPage" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@36b4fe2a depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.registrationConfirmationPage() on instance com.newtours.tests.BookFlightTest@36b4fe2a]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@36b4fe2a depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.registrationConfirmationPage() on instance com.newtours.tests.BookFlightTest@36b4fe2a]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flightDetailsPage -->
        <test-method signature="findFlightPage()[pri:0, instance:com.newtours.tests.BookFlightTest@36b4fe2a]" started-at="2023-05-11T15:30:26 GMT" name="findFlightPage" finished-at="2023-05-11T15:30:26 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.flightDetailsPage" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@36b4fe2a depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@36b4fe2a]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@36b4fe2a depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@36b4fe2a]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- findFlightPage -->
        <test-method signature="flightConfirmationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@36b4fe2a]" started-at="2023-05-11T15:30:26 GMT" name="flightConfirmationPage" finished-at="2023-05-11T15:30:26 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.findFlightPage" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.flightConfirmationPage() on instance com.newtours.tests.BookFlightTest@36b4fe2a depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@36b4fe2a]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.flightConfirmationPage() on instance com.newtours.tests.BookFlightTest@36b4fe2a depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@36b4fe2a]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flightConfirmationPage -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.newtours.tests.BookFlightTest@36b4fe2a]" started-at="2023-05-11T15:30:26 GMT" name="quitDriver" finished-at="2023-05-11T15:30:27 GMT" duration-ms="378" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
      </class> <!-- com.newtours.tests.BookFlightTest -->
    </test> <!-- book-flight-4 -->
  </suite> <!-- book-flight-module -->
</testng-results>
